docs/comment_copy_guidelines.md

```diff
@@ -0,0 +1,133 @@
+  kind of way. Meaning: showing how to do things, not discussing why. This might
+  mean that the official docs of the language don't suffice, as these are often
+  a code reference and don't show you how to use it and how it works. _However_,
+  you should provide a link only for more in-depth exploration. The person
+  should be able to understand what you mean from the answer directly, without
+  following the link.
+
+### Examples
+
+In JavaScript, a student has written a top-level constant with `let`. 
+
+```markdown
+<!-- not following these guidelines -->
+As you know, everyone uses const, you shouldn't use let or var.
+```
+
+This comment is not following these guidelines for the following reasons:
+
+- The action comes after the "explanation".
+- "As you know": we don't know if the student does.
+- "you shouldn't": you don't need the "you" to make this statement.
+- "everyone uses const": this isn't true _and_ can make the student feel as if
+  they did something horribly wrong.
+- Lacks an actual explanation of _why_ the advice is given.
+
+```markdown
+<!-- better -->
+Prefer `const` and `let` over `var`. The `const` declaration stops a variable 
+from being accidentally reassigned, which provides safety, and reduces 
+cognitive load for someone reading the code. [This article](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75) 
+explains the difference between the three.
+```
+
+In Go, a student has created a custom error instead of using the built-in ones:
+
+```markdown
+<!-- not following these guidlines -->
+I see you are creating a custom `error`. This is perfectly fine! If you did not
+know about `errors.New` and `fmt.Errorf` have a look at them as they are much 
+simpler ways to create an error. Custom errors are helpful if you want to check 
+if an error is of a certain type later.
+```
+
+This comment is not following these guidelines for the following reasons:
+
+- "I see": the analyzer is not a person: avoid I.
+- "This is perfectly fine!": Apparently it's not otherwise the reassuring wasn't
+  necessary. This probably can be left out all together; if you want to give a
+  generic tip about something existing, you can say exactly that: "An
+  alternative, equally valid way of doing x is y."
+- "If you did not know about": Leave this overly wordy word group out.
+
+```markdown
+<!-- better -->
+A custom `error` is typically used to provide custom behavior, or to distinguish
+on type later. For simpler cases, it's more common to rely on `errors.New` or 
+`fmt.Errorf`. This [in-depth article](https://golangbot.com/custom-errors/) about 
+custom errors might be interesting.
+```
+
+[git-website-copy]: https://github.com/exercism/website-copy/tree/master/automated-comments
+[issue-ci-comments]: https://github.com/exercism/automated-mentoring-support/issues/51
+[git-website-copy-label]: https://github.com/exercism/website-copy/pulls?q=is%3Aopen+is%3Apr+label%3Atype%2Fanalyzer-comments
```

Comment:
Should this be `analyzer-comments-prs` instead of `git-website-copy-label`?

Reply:
🤷‍♂ just a link label (never displayed) -- I don't mind to change it hehe 😁